starknet_py/__init__.py

```diff
@@ -0,0 +1,3 @@
+from marshmallow import EXCLUDE, RAISE, Schema
+
+Schema.Meta.unknown = EXCLUDE  # type: ignore
```

Comment:
just to be double sure - you have checked that this won't overwrite a user setting in script, right? I mean:
```
# import marshmallow
# set Meta to whatever
# import starknet.py
# the rest of the script
```

so the importing starknet.py (and thus this file as well), does not overwrite the user setting